newIDE/app/src/AssetStore/ExampleStore/ExampleInformationPage.js

```diff
@@ -0,0 +1,113 @@
+// @flow
+import { Trans } from '@lingui/macro';
+import * as React from 'react';
+import { type ExampleShortHeader } from '../../Utils/GDevelopServices/Example';
+import { isCompatibleWithAsset } from '../../Utils/GDevelopServices/Asset';
+import { MarkdownText } from '../../UI/MarkdownText';
+import Text from '../../UI/Text';
+import AlertMessage from '../../UI/AlertMessage';
+import { getIDEVersion } from '../../Version';
+import { Column, Line } from '../../UI/Grid';
+import { ColumnStackLayout, ResponsiveLineStackLayout } from '../../UI/Layout';
+import { ExampleThumbnailOrIcon } from './ExampleThumbnailOrIcon';
+import Window from '../../Utils/Window';
+import { UserPublicProfileChip } from '../../UI/User/UserPublicProfileChip';
+import { ExampleDifficultyChip } from '../../UI/ExampleDifficultyChip';
+import { ExampleSizeChip } from '../../UI/ExampleSizeChip';
+import {
+  isStartingPointExampleShortHeader,
+  getStartingPointExampleShortHeaderTitle,
+} from '../../ProjectCreation/EmptyAndStartingPointProjects';
+const isDev = Window.isDev();
+
+const styles = {
+  chipsContainer: {
+    display: 'flex',
+    flexWrap: 'wrap',
+  },
+};
+
+type Props = {|
+  exampleShortHeader: ExampleShortHeader,
+|};
+
+export const openExampleInWebApp = (exampleShortHeader: ExampleShortHeader) => {
+  Window.openExternalURL(
+    `${
+      isDev ? 'http://localhost:3000' : 'https://editor.gdevelop.io'
+    }/?create-from-example=${exampleShortHeader.slug}`
+  );
+};
+
+const getExampleName = (exampleShortHeader: ExampleShortHeader) => {
+  return isStartingPointExampleShortHeader(exampleShortHeader)
+    ? getStartingPointExampleShortHeaderTitle(exampleShortHeader)
+    : exampleShortHeader.name;
+};
+
+const ExampleInformationPage = ({ exampleShortHeader }: Props) => {
+  const isCompatible = isCompatibleWithAsset(
+    getIDEVersion(),
+    exampleShortHeader
+  );
+  const hasIcon = exampleShortHeader.previewImageUrls.length > 0;
+
+  return (
+    <ColumnStackLayout expand noMargin>
+      {!isCompatible && (
+        <AlertMessage kind="error">
+          <Trans>
+            Unfortunately, this example requires a newer version of GDevelop to
```

Comment:
Should this message be a bit more positive: "Download the latest version to check out this example!"

Reply:
Excellent idea.